src/quantum/README.rst

```diff
@@ -0,0 +1,14 @@
+Microsoft Azure CLI 'quantum' Extension
```

Comment:
pls include Azure CLI usage instruction in readme, example https://github.com/Azure/azure-cli-extensions/blob/master/src/blueprint/README.md

Reply:
Just like the documentation in email.

Reply:
Improving the README will be tackled in a different PR: 
https://github.com/anpaz-msft/azure-cli-extensions/pull/5

Reply:
Never mind. I have already incorporated the documentation. We'll still review it and may incorporate some changes later.